buku.py

```diff
@@ -3259,6 +3294,22 @@ def main():
         # Enable color in logs
         setup_logger(logger)
 
+    # Set colors
+    if args.colorstr:
+        ID_str = '%d. %s [%s]\n'
+        ID_DB_str = '%d. %s'
+        MUTE_str = '%s (L)\n'
+        URL_str = '%s   > %s\n'
+        DESC_str = '%s   + %s\n'
+        TAG_str = '%s   # %s\n'
+        colors = Colors(*[COLORMAP[c] for c in args.colorstr], reset=COLORMAP['x'])
+        ID_DB_str = colors.ID_DB_str + ID_DB_str + COLORMAP['x']
```

Comment:
The color for ID+title reflects in `buku -p` but not for the search results. Please fix.